ddcz/templates/postal_service/components/list.html

```diff
@@ -58,6 +58,11 @@
     {# Add icon later #}
     <section class="postal_letter_body">
         <section class="opponent_icon">
+            {% if letter.sender == user.userprofile.nick %}
+                <img src="{{ letter.receiver | nick_icon }}" alt="">
```

Comment:
Either don't put `alt` or fill it in—I think `Ikonka pro uživatele {{ letter.receiver }}` should be there, let's make it accessible